src/Core/Managed/Shared/Extensibility/TelemetryConfiguration.cs

```diff
@@ -133,6 +134,15 @@ public IList<ITelemetryInitializer> TelemetryInitializers
         }
 
         /// <summary>
+        /// Gets the list of <see cref="IMetricProcessor"/> objects used for custom metric data processing
+        /// before client-side metric aggregation process.
+        /// </summary>
+        public IList<IMetricProcessor> MetricProcessors
```

Comment:
I was changed to "processor" from "metric stream listener"to make it more consistent :) I'll leave it as is for now.